perfume-api/src/main/java/io/perfume/api/sample/infrastructure/api/SampleController.java

```diff
@@ -0,0 +1,60 @@
+package io.perfume.api.sample.infrastructure.api;
+
+
+import io.perfume.api.sample.application.SampleService;
+import io.perfume.api.sample.application.dto.SampleResult;
+import io.perfume.api.sample.infrastructure.api.dto.CreateSampleRequestDto;
+import io.perfume.api.sample.infrastructure.api.dto.SampleResponseDto;
+import io.perfume.api.sample.infrastructure.api.dto.UpdateSampleRequestDto;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.Min;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/samples")
+@RequiredArgsConstructor
+public class SampleController {
+
+    private final SampleService sampleService;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'member def modifier' has incorrect indentation level 4, expected level should be 2.